src/driver/sqlite/SqliteQueryRunner.ts

```diff
@@ -114,7 +109,7 @@ export class SqliteQueryRunner extends AbstractSqliteQueryRunner {
                 } else {
                     const result = new QueryResult()
 
-                    if (isInsertQuery) {
+                    if (isInsertQuery || isDeleteQuery) {
```

Comment:
can you please explain how does this change relate to the proposed RETURNING INTO feature?